src/Funds/FundForm/FundForm.js

```diff
@@ -284,6 +287,7 @@ const FundForm = ({
                       xs={3}
                     >
                       <FieldMultiSelection
+                        ariaLabelledBy="fund-form-transfer-from-label"
```

Comment:
but you're passing label there. what's the point of `ariaLabelledBy` with some magic value?

Reply:
Thanks. You're right. So will close this PR cause the issue is related to [STCOM-696](https://issues.folio.org/browse/STCOM-696)